hugegraph-rocksdb/src/main/java/com/baidu/hugegraph/backend/store/rocksdb/RocksDBStdSessions.java

```diff
@@ -1058,14 +1059,14 @@ private boolean match(int expected) {
         @SuppressWarnings("unused")
         private void dump() {
             this.seek();
-            System.out.println(">>>> scan from " + this.table + ": "  +
-                               (this.keyBegin == null ? "*" :
-                                StringEncoding.format(this.keyBegin)) +
-                               (this.iter.isValid() ? "" : " - No data"));
+            LOG.info(">>>> scan from {}: {}{}", 
+                    this.table,
+                    this.keyBegin == null ? "*" : StringEncoding.format(this.keyBegin),
+                    this.iter.isValid() ? "" : " - No data"));
```

Comment:
`" - No data"))` => `" - No data")`
there is an error: `/home/runner/work/incubator-hugegraph/incubator-hugegraph/hugegraph-rocksdb/src/main/java/com/baidu/hugegraph/backend/store/rocksdb/RocksDBStdSessions.java:1065:61: expecting RCURLY, found ')' -> [Help 1]`